### ISSUE TITLE
📝 Write an article about retrieving revision content

### ISSUE THREAD

**Issue by LucyMcGowan** [tidyverse/googledrive#219]
@benmarwick has agreed to do this based on conversation in https://github.com/tidyverse/googledrive/issues/218

**Comment by jennybc** [comment]
Is there still interest in this? A googledrive release is on the vague horizon. I'm making a pass through now, then will go back to gargle to push it out the door, then will return here.

**Comment by benmarwick** [comment]
Yes! I'm still interested to contribute, and I do have a draft of this that I think is nearly complete. Do you have some deadlines that I should aim for?

**Comment by jennybc** [comment]
Not really. This will be an article, so probably pkgdown only. So it's really not something that has to be drive by CRAN release rhythms. I was mostly interested in knowing whether to leave this open or not.

**Comment by nealmcb** [comment]
This sounds very valuable. Is the draft available for feedback?
Thanks!

**Comment by jennybc** [comment]
> And how might this be related to https://github.com/larsks/gitdriver?

This is my first expose to that. But perhaps its relevant for developing the article and, long-term, determining whether some similar functionality would be useful in googledrive more properly.

**Comment by mkiang** [comment]
@benmarwick just an FYI that you may want to use a document with enough revisions that it requires pagination. I've been running into pagination errors with the revision endpoint.
